packages/interface/src/components/complementary-area/style.scss

```diff
@@ -12,6 +12,9 @@
 
 	.components-panel {
 		border: none;
+		// Make a stacking context that keeps all descendents behind the sticky header
+		position: relative;
+		z-index: -1 + z-index(".components-panel__header");
```

Comment:
Based on other rules I think we're adding z-index entries to the flat _z-index.scss map. For folks 👀 please correct me otherwise.

This one's a little interesting. I understand the intent here is to be one less than the panel header since this element is a sibling (they should have a shared parent stacking context). The z-index does get set to 0 in the css file:

<img width="304" alt="Screen Shot 2021-02-11 at 2 13 41 PM" src="https://user-images.githubusercontent.com/1270189/107705435-92828c00-6c73-11eb-9f9c-2a9eaed046c4.png">

We could maybe do this in few ways:
1. Keep the expression as is
2. Add a simple entry to the z-index map, and leave a comment next to the [components-panel__header entry](https://github.com/WordPress/gutenberg/blob/2c56f53e1cae13008dab97335fb3cb0782cc9efe/packages/base-styles/_z-index.scss#L19) to update this if the values change. `".interface-complemtary-area .components-panel" : 0`. 
3. Enforce this in _z-index.scss (though note that SASS maps are immutable).

Reply:
4. Use bare integers for these rules and remove the entry for "components-panel__header" from the z-index map.

These elements are in a stacking context (of `.interface-interface-skeleton__sidebar`) with no other siblings so it would okay. Just have to check that there's no other rule elsewhere in the codebase coopting the z-index map entry for "components-panel__header" (I found none).

I'm all for removing stuff from that map but am fine with any of those options.

Reply:
@gwwar I have no strong preference for any of these. If I had to pick one I would add it to the map of values rather than computing it on the fly here, with an explanatory comment. But I think what's done here is fine with me as well, considering it's a "relative" z-index rather than an "absoute" one.